pkg/gpu/nvidia/manager_test.go

```diff
@@ -169,7 +169,7 @@ func Test_nvidiaGPUManager_Envs(t *testing.T) {
 		},
 		{
 			name:           "MPS enabled, single GPU request",
-			totalMemPerGPU: 80 * 1024,
+			totalMemPerGPU: 80 * 1024 * 1024 * 1024,
```

Comment:
Add a comment for the total limit (80GB)?

Reply:
Done